app/database/schema.prisma

```diff
@@ -60,6 +60,7 @@ model User {
   receivedInvites   Invite[]           @relation("invitee")
   teamMembers       TeamMember[]
   userOrganizations UserOrganization[]
+  Booking           Booking[]
```

Comment:
Can we please change the capitalization. For all the column names we use small letter.